NOVAthesisFiles/Schools/uminho/uminho-defaults.ldf

```diff
@@ -260,7 +260,7 @@
 }
 
 % Date
-\ntaddtocover[vspace=1,halign=l]{1-1,2-1}{%
+\ntaddtocover[vspace=1cm,halign=l]{1-1,2-1}{%
```

Comment:
Oh!  This suggestion is wrong.  That is not supposed to be `1cm` but `1` as it is.  The number without a unit means `\vfill`, which is great for relative vertical spacing.  If you replace it with `1cm` it will be out of format.